tests/tasks/setup_ipa.yml

```diff
@@ -53,6 +53,7 @@
   rescue:
     - name: FAILURE - check entropy
       command: cat /proc/sys/kernel/random/entropy_avail
+      changed_when: false
```

Comment:
don't worry about these - we'll change these later when we make all of the roles work with ansible-lint 6.x - looks like we have some ansible-core 2.14 work to do as well - `warn: false` breaks with 2.14